client/test/integration/markLogicIntegrationTestHelper.ts

```diff
@@ -62,7 +64,9 @@ export class IntegrationTestHelper {
     private clientDefaults = {
         host: this.hostname,
         port: this.port,
+        restPath: this.restPath,
```

Comment:
Wondering if this is supposed to be "restBasePath" - I didn't find any references to "restPath" in the rest of the PR.

Reply:
fixed